src/Entity/IdeasWorkshop/Need.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace AppBundle\Entity\IdeasWorkshop;
+
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use AppBundle\Entity\EntityEnabledTrait;
+use AppBundle\Entity\EntityNameSlugTrait;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(
+ *     uniqueConstraints={
+ *         @ORM\UniqueConstraint(name="need_slug_unique", columns="slug")
+ *     }
+ * )
+ *
+ * @UniqueEntity("name")
+ *
+ * @Algolia\Index(autoIndex=false)
+ */
+class Need
+{
+    use EntityNameSlugTrait;
+    use EntityEnabledTrait;
+
+    /**
+     * @var int
+     *
+     * @ORM\Column(type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    public function __construct(string $name, bool $enable = false)
+    {
+        $this->setName($name);
```

Comment:
Same here, you can set directly the property.

Reply:
:heavy_check_mark: